storybookjs/storybook
# Remove duplicate of the State plugin in the docs

Issue: There was a duplicate of the State plugin community plugins list in the docs

## What I did
I removed the duplicate

## How to test

- Is this testable with Jest or Chromatic screenshots? n/a
- Does this need a new example in the kitchen sink apps? n/a
- Does this need an update to the documentation? n/a

If your answer is yes to any of these, please make sure to include it in your PR.
